examples/tutorials/curation/plot_1_automated_curation.py

```diff
@@ -0,0 +1,241 @@
+"""
+Model Based Curation Tutorial
+=============================
+
+This notebook provides a step-by-step guide on how to use a machine learning classifier for
+curating spike sorted output. We'll download a toy model and use it to label our sorted data by using
+Spikeinterface. We start by importing some packages
+"""
+
+import warnings
+warnings.filterwarnings("ignore")
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import spikeinterface.full as si
```

Comment:
I would import by module here, since it wil clearly show where the different functions are coming from